SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2465,13 +2479,14 @@ \subsection{Alignment}
 
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
-InputPixelType  m_ForegroundValue;
-OutputPixelType m_BackgroundValue;
-
-unsigned int m_MaximumIterations;
-
-std::vector<double> m_Sensitivity;
-std::vector<float>  m_Overlaps;
+bool                                      m_Stop{ false };
+double                                    m_Value{ 0.0 };
+StopConditionGradientDescentOptimizerEnum m_StopCondition{
+  StopConditionGradientDescentOptimizerEnum::MaximumNumberOfIterations
+};
+SizeValueType      m_NumberOfIterations{ 100 };
```

Comment:
Now there is no empty line, but alignment changes here.

Reply:
I guess that was decided by `clang-format` in the code base, but I agree, bad choice of mine again. Fixed in 88ea969.